Redirecting user to the correct information (#5626)
MicrosoftDocs/windows-itpro-docs: devices/surface/considerations-for-surface-and-system-center-configuration-manager.md

## Support for Surface Pro X
Beginning in version 1802, Endpoint Configuration Manager includes client management support for Surface Pro X. Note however that running the Endpoint Configuration Manager agent on Surface Pro X may accelerate battery consumption. In addition, operating system deployment using Endpoint Configuration Manager is not supported on Surface Pro X. For more information, refer to:

> - [What's new in version 1802 of System Center Configuration Manager] (https://docs.microsoft.com/configmgr/core/plan-design/changes/whats-new-in-version-1802)

Reviewer: ```suggestion
- [What's new in version 1802 of System Center Configuration Manager](https://docs.microsoft.com/configmgr/core/plan-design/changes/whats-new-in-version-1802)
```

- [Deploying, managing, and servicing Surface Pro X](surface-pro-arm-app-management.md)

## Updating Surface device drivers and firmware

> For devices that receive updates through Windows Update, drivers for Surface components – and even firmware updates – are applied automatically as part of the Windows Update process. For devices with managed updates, such as those updated through Windows Server Update Services (WSUS), or SCCM see [https://docs.microsoft.com/surface/manage-surface-driver-and-firmware-updates/](Manage Surface driver and firmware updates).

Reviewer: ```suggestion
For devices that receive updates through Windows Update, drivers for Surface components – and even firmware updates – are applied automatically as part of the Windows Update process. For devices with managed updates, such as those updated through Windows Server Update Services (WSUS) or SCCM, see [Manage Surface driver and firmware updates](https://docs.microsoft.com/surface/manage-surface-driver-and-firmware-updates/).
```

## Updating Surface device drivers and firmware

> For devices that receive updates through Windows Update, drivers for Surface components—and even firmware updates—are applied automatically as part of the Windows Update process. For devices with managed updates, such as those updated through Windows Server Update Services (WSUS) or SCCM, see [Manage Surface driver and firmware updates](https://docs.microsoft.com/surface/manage-surface-driver-and-firmware-updates/).

Reviewer 2: From a grammatical and readability point of view, it is useful to avoid phrases containing the word "that" (if possible), because it tends to add fluff and no logic to the reader's understanding.
```suggestion
For devices receiving updates through Windows Update, drivers for Surface components—and even firmware updates—are applied automatically as part of the Windows Update process. For devices with managed updates, such as those updated through Windows Server Update Services (WSUS) or SCCM, see [Manage Surface driver and firmware updates](https://docs.microsoft.com/surface/manage-surface-driver-and-firmware-updates/).
```
Other than that, this PR looks quite good to me.

Reviewer 2: (By saying "it is useful to avoid phrases containing 'that' ", I mean "when it is used as a **relative pronoun**, referencing something mentioned just before in the same sentence".)
